### ISSUE TITLE
DVH scaling Volume, not Dose

### ISSUE THREAD

**Issue by cvelten** [dicompyler/dicompyler-core#300]
(dicompyler-core=0.5.5)

In `dvh.py` the DVH object is created with the following return
```
        return cls(counts=data[1::2] * dvh.DVHDoseScaling,
                   bins=data[0::2].cumsum(),
```
With the counts being volume or "counts" data, then the `DVHDoseScaling` should rather be applied to the `bins`.

This is a sample output, the doses and volume are wrong by a factor of 10 (the DVHDoseScaling is 0.01), verified with TPS and other 3rd party software.
<img width="830" alt="image" src="https://user-images.githubusercontent.com/7443659/178582094-ed48c928-f46b-46c0-881a-5a4cbde70cfc.png">

**Comment by bastula** [comment]
Thanks for submitting this issue and the related fix in #301. I don't think I ever ran across a DICOM DVH with a scaling factor other than 1.0, hence why this issue never came up.

Might be worth to have a unit test to check that DVHDoseScaling works as expected if the value changes.

**Comment by bastula** [comment]
Added unit test in [b59c31b](https://github.com/dicompyler/dicompyler-core/commit/b59c31bfb230f666e71e5438e23fe65613c377db).
